mods/webui/src/stories/chip/Chip.tsx

```diff
@@ -0,0 +1,15 @@
+import  { useEffect, useState } from "react";
```

Comment:
Would you remove `useEffect` and `useState` as they are not being used?

Reply:
done.